tests/files/fail/9f.yaml

```diff
@@ -33,4 +33,5 @@ schema:
             - type: str
               unique: True
 errors:
+  - 'value.notunique :: value: foo : /2/name : /1/name'
```

Comment:
Here is your problem, you have written `foo` here when you mean `bar`

I have the following 

```
errors:
  - 'value.notunique :: value: foo : /0/groups/3 : /0/groups/0'
  - 'value.notunique :: value: bar : /2/name : /1/name'
```

and it works fine locally for me